src/main/java/io/omnition/loadgenerator/util/ScheduledTraceGenerator.java

```diff
@@ -22,14 +22,31 @@
     private final String route;
     private final ITraceEmitter emitter;
 
+    public enum LogLevel {
+        Minimum,
+        Verbose
+    }
+
+    private final LogLevel logLevel;
+    private final SummaryLogger summaryLogger;
+
     public ScheduledTraceGenerator(
-            Topology topology, String service, String route, int tracesPerHour, ITraceEmitter emitter) {
+            Topology topology,
+            String service,
+            String route,
+            int tracesPerHour,
+            ITraceEmitter emitter,
+            LogLevel logLevel,
```

Comment:
I don't even think we need a verbose mode, this is a relic from the time when we were sending a low volume of load
you might even make the minimum level that dumps details every 1 minute

Reply:
If I read Steve's comment right verbose may be actually needed: https://github.com/Omnition/synthetic-load-generator/pull/28#issuecomment-487662416
@flands can you clarify?